### PR TITLE
cube puzzle

### PR DESCRIPTION
**Pull request by marrable**
This is the form input for the physical 3D printed cube puzzle. Will be in touch to confirm the final format of the domain + endpoint for the QR code incorporation into the CAD model to print.

### CODE REVIEW

File: openday_scavenger/puzzles/cube/views.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+@router.get("/")
+@router.get("")
+async def index(
+    request: Request,
+    db: Annotated[Session, Depends(get_db)],
+    visitor: Annotated[VisitorAuth, Depends(get_auth_visitor)],
+):
+    state = get_puzzle_state(db, puzzle_name=PUZZLE_NAME, visitor_auth=visitor)
+    state["state_access_count"] = state.get("state_access_count", 0) + 1
+    set_puzzle_state(db, puzzle_name=PUZZLE_NAME, visitor_auth=visitor, state=state)
```
Comment (stmudie) [inline openday_scavenger/puzzles/cube/views.py]: These should be removed. Puzzle state exists for puzzles that need to store an intermediate state before submission. No need to use the demo code here.

Comment (marrable) [reply openday_scavenger/puzzles/cube/views.py]: Oh good catch

Comment (marrable) [reply openday_scavenger/puzzles/cube/views.py]: Removed those lines and associated imports.

**Review by stmudie — APPROVED** [review]
Very cool. Thanks!!

**Comment by pcsmomo** [comment]
Hi Sam! 👋

**Pull request merged**